spec/features/categories/user_can_create_categories_with_a_form_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "User can create a new Category by filling out a form" do
+  scenario "user fills out a category form" do
```

Comment:
don't forget to test the sad path here. what kinds of errors should a user see and what page should they be on if they enter invalid data?